packages/common/src/hooks/useSortedData/useSortedData.ts

```diff
@@ -1,33 +1,8 @@
-import { SortRule, SortBy } from '@openmsupply-client/common';
+import { SortRule, SortBy, getDataSorter } from '@openmsupply-client/common';
 import { ObjectWithStringKeys } from './../../types/utility';
 import { useEffect, useState } from 'react';
 import { useSortBy } from '../useSortBy';
 
-const parseValue = (object: any, key: string) => {
-  const value = object[key];
-  if (typeof value === 'string') {
-    const valueAsNumber = Number.parseFloat(value);
-
-    if (!Number.isNaN(valueAsNumber)) return valueAsNumber;
-    return value.toUpperCase(); // ignore case
-  }
-  return value;
-};
-
-const getDataSorter = (sortKey: any, desc: boolean) => (a: any, b: any) => {
-  const valueA = parseValue(a, sortKey);
-  const valueB = parseValue(b, sortKey);
-
-  if (valueA < valueB) {
-    return desc ? 1 : -1;
-  }
-  if (valueA > valueB) {
-    return desc ? -1 : 1;
-  }
-
-  return 0;
-};
-
```

Comment:
I think I have copy pasted this in about 12 places. So I put them all in `common/utils/arrays/sorters`